### DISCUSSION TITLE
[Vote ended on 2023-02-08] Ansible docsite personas

### DISCUSSION THREAD

**Discussion by oraNod** [ansible-community/community-topics#189] [Votes done]
The purpose of this vote is to ensure that the Ansible community agrees to proceed with docsite personas in this topic: https://github.com/ansible-community/community-topics/issues/175

Personas are available at: https://hackmd.io/pZb5w5JFRQW3RJ73n23tlw?both#/

Personas are a mechanism to map user journeys on `docs.ansible.com` and provide better retrievability and navigation to community content.

The purpose of this vote is to ensure that the Ansible community is comfortable with the docsite personas we have identified and we have a reasonable consensus for moving forward with docsite improvements based on those personas. "Please tell us what you think about the personas and if you see any improvements or have objections."

The vote is *NOT* intended to finalize discussion around personas or set them in stone. We can continue to develop, modify, and add new personas after the vote takes place. 

After the vote we will use the personas to make progress on changes to the Ansible docsite "docs.ansible.com" and will present those to the community for separate consideration.

**Comment by Andersson007** [comment]
+1 for proceeding with docsite personas provided that there will be another vote when we find a final consensus on them

**Reply by oraNod** [reply]
Is it a bit premature to put this to a vote? We thought by creating a vote we could essentially get community "acks" on the personas.

If you, or others, feel we need to find a final consensus before voting then maybe we should hold off?
